tests/nightly/test_large_array.py

```diff
@@ -37,6 +37,7 @@
 LARGE_X = 100000000
 SMALL_X = 100
 SMALL_Y = 50
+LARGE_SQ_X = 80000
```

Comment:
Can we make this  70k instead that will still lead to 4.9 Billion elements and still greater then 4.3 Billion elements. This way it will take little less time

Reply:
done!